frappe/integrations/doctype/dropbox_settings/dropbox_settings.py

```diff
@@ -131,12 +131,11 @@ def upload_from_folder(path, is_private, dropbox_folder, dropbox_client, did_not
 
 	for f in frappe.get_all("File", filters={"is_folder": 0, "is_private": is_private,
 		"uploaded_to_dropbox": 0}, fields=['file_url', 'name', 'file_name']):
-		if is_private:
-			filename = f.file_url.replace('/private/files/', '')
+		if not f.file_url:
+			filename = f.filename.rsplit('/',1)[-1]
```

Comment:
```suggestion
			filename = f.filename
```
why is `rsplit` required incase of `f.filename`?

Reply:
Because while I was finding the cause of this issue I found a few filenames were actually paths hence using rsplit because if there is a path in the filename it will get the name if there isn't then too it gets filename.